pkgs/by-name/un/unimatrix/package.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, python3
+, fetchFromGitHub
+, git
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
+    # Needed for setuptools-scm to detect version
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools-scm
+    python3.pkgs.pip
```

Not entirely sure about this, but I believe python3Packages is the canonical way of referring to the python package set.

```suggestion
, python3Packages
, fetchFromGitHub
, git
}:

python3Packages.buildPythonApplication rec {
  pname = "unimatrix";
  version = "unstable-2023-04-25";

  src = fetchFromGitHub {
    owner = "will8211";
    repo = pname;
    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
    # Needed for setuptools-scm to detect version
    leaveDotGit = true;
  };

  nativeBuildInputs = [
    python3Packages.setuptools-scm
    python3Packages.pip
```

---

Done